lib/redis.js

```diff
@@ -72,16 +73,17 @@ class RedisClient {
 	}
 
 	attachEvent() {
-		const rl = readline.createInterface(process.stdin, process.stdout);
+		this.rl = readline.createInterface(process.stdin, process.stdout);
```

Comment:
@lujiajing1126 why do yo need rl on this?

Reply:
Previously, I want to use some other `print` methods...
I could revert this change.